pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -287,7 +287,7 @@ type AlertQueryExport struct {
 
 type RelativeTimeRangeExport struct {
 	FromSeconds int64 `json:"from" yaml:"from" hcl:"from"`
-	ToSeconds   int64 `json:"to" yaml:"to" hcl:"to"`
+	ToSeconds   int64 `json:"to,omitempty" yaml:"to,omitempty" hcl:"to"`
```

Comment:
The OpenAPI Spec has this as not required (= the generated go client has it as `omitempty`) so I'm reflecting the change here